include/grpc/grpc.h

```diff
@@ -351,6 +351,23 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
+/** Propagate census context */
+#define GRPC_PROPAGATE_CENSUS_CONTEXT ((gpr_uint32)2)
```

Comment:
I don't suppose you'd want to send me a PR against this one implementing
it? :)

On Mon, Aug 3, 2015 at 9:36 AM Alistair Veitch <notifications@github.com>
wrote:

> In include/grpc/grpc.h
> <https://github.com/grpc/grpc/pull/2750#discussion_r36103509>:
>
> > @@ -351,6 +351,23 @@ typedef struct grpc_op {
> >    } data;
> >  } grpc_op;
> >
> > +/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
> > + * grpc_call */
> > +/** Propagate deadline */
> > +#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
> > +/** Propagate census context */
> > +#define GRPC_PROPAGATE_CENSUS_CONTEXT ((gpr_uint32)2)
>
> SGTM.
>
> Could we find a half-way point perhaps?
>
> GRPC_PROPAGATE_STATS
> GRPC_PROPAGATE_TRACING
>
> —
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2750/files#r36103509>.
>